templates/default/testconnector-launch.sh.erb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+help() {
+    echo ""
+    echo "usage: $0 [input_json_filepath image_name]"
+    echo ""
+    exit 1
+}
+
+if [ "$#" != "2" ] || [ "$1" == "-h" ] ; then
+  help
+fi
+
+REGISTRY_URL='<%= @registry_addr %>'
```

Comment:
This won't work in the cloud where each user will have its own ECR/ACR docker registry.
We should instead do the same as in Git where the full docker image name is passed to the script 
https://github.com/logicalclocks/hopsworks-ee/blob/master/hopsworks-common/src/main/java/io/hops/hopsworks/common/git/util/GitContainerArgumentsWriter.java#L106

Reply:
fixed